client/components/AIAssistantPanel.tsx

```
@@ -234,16 +243,20 @@ export function AIAssistantPanel({
 
   return (
     <View
       style={[styles.container, { backgroundColor: theme.backgroundDefault }]}
     >
       {/* Header */}
       <View style={[styles.header, { borderBottomColor: theme.border }]}>
-        <Text style={[styles.headerTitle, { color: theme.textDefault }]}>
-          🤖 AI Assistant
-        </Text>
+        <View style={styles.headerTitleRow}>
+          <AppIcon name="cpu" size={20} color={theme.primary} testID="ai-assistant-header-icon" />
```

ESLint

[error] Replace `·name="cpu"·size={20}·color={theme.primary}·testID="ai-assistant-header-icon"` with `⏎············name="cpu"⏎············size={20}⏎············color={theme.primary}⏎············testID="ai-assistant-header-icon"⏎·········`

(prettier/prettier)